index.js

```diff
@@ -1,68 +1,225 @@
+const profileInfoSubtitle = document.querySelector(`.${CLASSES.PROFILE_INFO_SUBTITLE}`);
+const profileInfoEditButton = document.querySelector(`.${CLASSES.PROFILE_INFO_EDIT_BUTTON}`);
+const profileAddButton = document.querySelector(`.${CLASSES.PROFILE_ADD_BUTTON}`);
 
-formCloseIcon.addEventListener(EVENTS.CLICK, (evt) => {
-    evt.preventDefault();
-    formElement.classList.remove('editForm_opened');
-});
+const editForm = document.querySelector(`.${CLASSES.EDIT_FORM}`);
+const editFormButton = document.querySelector(`.${CLASSES.EDIT_FORM_BUTTON}`);
+const editFormCloseIcon = document.querySelector(`.${CLASSES.EDIT_FORM_CLOSE_ICON}`);
 
-formButton.addEventListener(EVENTS.CLICK, (evt) => { 
-    evt.preventDefault();
-    infoTitle.textContent = nameInput.value;
-    infoSubtitle.textContent = jobInput.value;
-    formElement.classList.remove('editForm_opened');
-});
+const addForm = document.querySelector(`.${CLASSES.ADD_FORM}`);
+const addFormButton = document.querySelector(`.${CLASSES.ADD_FORM_BUTTON}`);
+const addFormCloseIcon = document.querySelector(`.${CLASSES.ADD_FORM_CLOSE_ICON}`);
+
+const imagePopup = document.querySelector(`.${CLASSES.IMAGE_POPUP}`);
+const imagePopupContainer = document.querySelector(`.${CLASSES.IMAGE_POPUP_CONTAINER}`);
+const imagePopupImage = document.querySelector(`.${CLASSES.IMAGE_POPUP_IMAGE}`);
+const imagePopupTitle = document.querySelector(`.${CLASSES.IMAGE_POPUP_TITLE}`);
+const imagePopupCloseIcon = document.querySelector(`.${CLASSES.IMAGE_POPUP_CLOSE_ICON}`);
+
+const elements = document.querySelector(`.${CLASSES.ELEMENTS}`);
+const element = document.querySelector(`.${CLASSES.ELEMENT}`);
+const elementImage = document.querySelector(`.${CLASSES.ELEMENT_IMAGE}`);
+const elementText = document.querySelector(`.${CLASSES.ELEMENT_TEXT}`);
+const elementLikeStatus = document.querySelector(`.${CLASSES.ELEMENT_LIKE_STATUS}`);
+const elementTrashButton = document.querySelector(`.${CLASSES.ELEMENT_TRASH_BUTTON}`);
 
+const initialCards = [
+    {
+      name: 'Архыз',
+      link: 'https://pictures.s3.yandex.net/frontend-developer/cards-compressed/arkhyz.jpg'
+    },
+    {
+      name: 'Челябинская область',
+      link: 'https://pictures.s3.yandex.net/frontend-developer/cards-compressed/chelyabinsk-oblast.jpg'
+    },
+    {
+      name: 'Иваново',
+      link: 'https://pictures.s3.yandex.net/frontend-developer/cards-compressed/ivanovo.jpg'
+    },
+    {
+      name: 'Камчатка',
+      link: 'https://pictures.s3.yandex.net/frontend-developer/cards-compressed/kamchatka.jpg'
+    },
+    {
+      name: 'Холмогорский район',
+      link: 'https://pictures.s3.yandex.net/frontend-developer/cards-compressed/kholmogorsky-rayon.jpg'
+    },
+    {
+      name: 'Байкал',
+      link: 'https://pictures.s3.yandex.net/frontend-developer/cards-compressed/baikal.jpg'
+    },
+  ]; 
+
+const renderCard = () => {
+    let elements = document.querySelector('.elements');
+        elements.innerHTML = '';
```

Comment:
отступы поехали, либо ты так решил оформить. но с точки зрения чтения кода это не правильно, тк отступ говорит о вложенности, но тут ее нет. Начинай новое действие с новой строки с одинаковым отступом. Больший отступ делается когда появлется вложенность в виде функций, циклов, скобок.
```suggestion
    let elements = document.querySelector('.elements');
    elements.innerHTML = '';
```